core/meterer/dynamodb_metering_store_test.go

```diff
@@ -47,11 +46,6 @@ func setupTest(t *testing.T) *testContext {
 	err = meterer.CreateGlobalReservationTable(clientConfig, tc.globalBinTable)
 	require.NoError(t, err)
 
-	// Register cleanup to remove tables after test completes
-	t.Cleanup(func() {
```

Comment:
Wouldn't this affect the other tests that depend on it?

Reply:
none of the current tests depend on each other at the moment; they will each get a setup and cleanup. i think it is generally better to keep each tests isolated

Reply:
But wouldn't a test like `TestGetLargestCumulativePayment` no longer get cleaned up after it runs?

Reply:
I see, added cleanup to all the tests in this file